_posts/neatoskeeto/2017-05-22-matt-turtle-hack.md

```diff
@@ -0,0 +1,37 @@
+--- 
+layout: post
+author: neatoskeeto
+title: "Matt's TurtleHack Inspired by Oz"
+---
+
+
+<iframe src="https://trinket.io/embed/python/c424cb87eb" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+
+I started out doing a grid along the x and y axes, but then I noticed that the grid lines resembled prison bars. I then decided to go with a prison theme based on my favorite tv show Oz.
+I used guess and check to make the eyes look like they are in the right place. It took me a long time to make the mouth, which was just a bunch of guessing and checking. Making the tears was really difficult. I first used the stamp function to creat arrowhead-shaped tears, but I accidentally created a trail of blue behind it because I forgot to do the penup function. I thought the trail looked better. 
+Later, I tried to make the pen larger, and I realized that looked much better, so I erased the stamps and just made lines of increasing width.
+I wasn't sure how the tears were oriented, that is, if they are going perfectly straight down or not. I can't tell, and I would like to make it so that they face straight down. I'm sure there's an easy way to do that.
+I would also like to make it look like he's behind a prison door, instead of the bars floating in nothing. 
```

Comment:
Perhaps a background color, with a light rectangle fill on the bars outline? You'd need to make sure you made the bars background before anything else that'll be on top of it.